ocaml/typing/typedecl.mli

```diff
@@ -47,7 +47,14 @@ val transl_with_constraint:
     Typedtree.type_declaration
 
 val abstract_type_decl:
```

Comment:
Not new in this patch, but this function isn't used outside Typedecl. Remove from mli?

Reply:
I push back weakly, as upstream has not removed it.